src/containers/download-manager/DownloadManagerScreen.tsx

```diff
@@ -1,15 +1,12 @@
-import { Component, type ReactElement } from 'react';
 import DownloadManager from '../../components/downloadManager/DownloadManagerDashboard';
 import ErrorBoundary from '../../components/util/ErrorBoundary';
 
-class DownloadManagerScreen extends Component {
-  render(): ReactElement {
-    return (
-      <ErrorBoundary>
-        <DownloadManager {...this.props} />
-      </ErrorBoundary>
-    );
-  }
-}
+const DownloadManagerScreen = ({ ...props }) => {
+  return (
+    <ErrorBoundary>
+      <DownloadManager {...props} />
+    </ErrorBoundary>
+  );
+};
```

Comment:
Finally started migrating some class components to functional ones :tada: 

I verified that these load fine in the application.